### PR TITLE
Refine FeedbackQuestionSubmissionEdit Slice

### PR DESCRIPTION
**Pull request by MrFool**
Refinements include:

1. Indentation errors (according to coding style guidelines)
2. Stricter equality checks
3. Missing / Additional Spacing errors (according to coding style guidelines)
4. Quotation errors (according to coding style guidelines)
5. Consistent use of spaces rather than mix of tabs and spaces
6. Single line if-blocks without bracers (disallowed according to coding style guidelines)
7. Incorrect line breakages regarding operators (according to coding style guidelines)
8. jQuery wrapped objects prefixed with $
9. Java method declaration ([public protected private] [static] ... according to coding style guidelines)
10. Some typo fixes
11. Improved testing code coverage
12. Removed redundant html test(s)
13. Removed redundant testing methods (unused in the relevant page)

Java Side
. Assumption.java
. Action.java
. FeedbackQuestionData.java
. FeedbackQuestionsDb.java
. FeedbackQuestionSubmissionEditPageAction.java
. FeedbackQuestionSubmissionEditSaveAction.java
. FeedbackQuestionSubmissionEditPageData.java

JavaScript Side
. common.js
. feedbackSubmissionsEdit.js
. instructor.js

Java Servlet Side
. instructorFeedbackQuestionSubmissionEdit.jsp

Testing Side
. InstructorFeedbackQuestionSubmissionEditPageActionTest.java
. InstructorFeedbackQuestionSubmissionEditSaveActionTest.java
. InstructorFeedbackQuestionSubmitPageUiTest.java
. FeedbackQuestionSubmitPage.java

### CODE REVIEW

**Comment by damithc** [comment]
Don't forget the test cases. use a coverage tool to check coverage. Refer to our coding guidelines and testing guidelines for more info.

**Comment by MrFool** [comment]
Yupp, dev green atm but have not done the coverage check so I'll do that

**Comment by MrFool** [comment]
I see you mean I have not refined the tests, I misunderstood, I'll look into that as well.

**Comment by MrFool** [comment]
Updated commits and original post with testing refinements and summary

File: src/test/java/teammates/test/cases/ui/browsertests/InstructorFeedbackQuestionSubmitPageUiTest.java [hunk 1]
```diff
@@ -114,17 +112,15 @@ private void testSubmitAction() {
                 BackDoor.getFeedbackResponse(fq.getId(),
                     "IFQSubmitUiT.instr@gmail.tmt",
                     "IFQSubmitUiT.instr@gmail.tmt").getResponseDetails().getAnswerString());
-        
-        submitPage.verifyHtmlMainContent("/instructorFeedbackQuestionSubmitPageFilled.html");
```
Comment (kanghj) [inline src/test/java/teammates/test/cases/ui/browsertests/InstructorFeedbackQuestionSubmitPageUiTest.java]: Why is this removed?

File: src/test/java/teammates/test/cases/ui/InstructorFeedbackQuestionSubmissionEditPageActionTest.java [hunk 2]
```diff
@@ -31,6 +31,7 @@ public static void classSetUp() throws Exception {
 
     @Test
     public void testExecuteAndPostProcess() throws Exception {
+        @SuppressWarnings("unused")
         AccountAttributes instructorAccount = dataBundle.accounts.get("instructor1OfCourse1");
```
Comment (kanghj) [inline src/test/java/teammates/test/cases/ui/InstructorFeedbackQuestionSubmissionEditPageActionTest.java]: If this is not used, then we can just remove it.

File: src/test/java/teammates/test/cases/ui/browsertests/InstructorFeedbackQuestionSubmitPageUiTest.java (hunk 1 above)
Comment (MrFool) [inline src/test/java/teammates/test/cases/ui/browsertests/InstructorFeedbackQuestionSubmitPageUiTest.java]: From looking through, the backdoor accesses are already checking for the changes expected, there doesn't seem to be a need for the HTML verification tests unless it was meant for testing something else specific?

File: src/test/java/teammates/test/cases/ui/InstructorFeedbackQuestionSubmissionEditPageActionTest.java (hunk 2 above)
Comment (MrFool) [inline src/test/java/teammates/test/cases/ui/InstructorFeedbackQuestionSubmissionEditPageActionTest.java]: It is not used until later on when instructorAccount is changed to something else, only the initial instantiation is not used if I'm not wrong

Comment (kanghj) [inline src/test/java/teammates/test/cases/ui/InstructorFeedbackQuestionSubmissionEditPageActionTest.java]: Hmm, I don't see it used anywhere in the file.

Comment (MrFool) [inline src/test/java/teammates/test/cases/ui/InstructorFeedbackQuestionSubmissionEditPageActionTest.java]: You're right, it was changed but it is never used, I was mistaken and thought that it was changed and used, removing it in the coming commit.

**Comment by MrFool** [comment]
Need to deconflict one file after recent merges, changing back to ongoing
